phase1/vsphere/README.md

```diff
@@ -48,16 +49,141 @@ cd kubernetes-anywhere
 ### Enter the `kubernetes-anywhere` deployment environment:
 
 ```shell
-make docker-dev
+make docker-dev # The first run of this will build the docker image to be used and can take a few minutes.
 ```
 
 ### Start the deployment wizard:
 
 ```shell
 make deploy
 ```
+
 and fill complete the config wizard to deploy a kubernetes-anywhere cluster.
 
+Sample config below
+
+* Select the number of nodes. Master + Number of nodes will be deployed.
+```
+*
+* Phase 1: Cluster Resource Provisioning
+*
+number of nodes (phase1.num_nodes) [4] (NEW) 8
+```
+
+* Set the cluster name. A folder with the cluster name will be created to place all the VMs.
+```
+cluster name (phase1.cluster_name) [kubernetes] (NEW) k8s
+```
+
+* Select the provider
+```
+*
+* cloud provider: gce, azure or vsphere
+*
+cloud provider: gce, azure or vsphere (phase1.cloud_provider) [gce] (NEW) vsphere
+```
+
+* Set the vCenter URL
+```
+  *
+  * vSphere configuration
+  *
+  vCenter URL Ex: 10.192.10.30 or myvcenter.io (phase1.vSphere.url) [] (NEW) 10.192.72.70
+```
+
+* Set the port for vCenter communication. Unless vCenter is setup with a different port select the default port.
+```
+  vCenter port (phase1.vSphere.port) [443] (NEW) 
+```
+
+* Set the user name for vCenter. All vCenter operations will be performed using these user credentials
+```
+  vCenter username (phase1.vSphere.username) [] (NEW) administrator@myvcenter.io
+```
+
+* Set the password for vCenter.
+```
+  vCenter password (phase1.vSphere.password) [] (NEW) MyPassword#3
+```
+
+* Set the type of certificate used by vCenter. Set to true for self signed certificates
+```
+  Does host use self-signed cert (phase1.vSphere.insecure) [true] (NEW) 
+```
+
+* Set the datacenter in vCenter to use
+```
+  Datacenter (phase1.vSphere.datacenter) [datacenter] (NEW) PA-DC
```

Comment:
I think it would be useful to upload snippet of vCenter and refer to values in that snippet.